quickstarts/aws/amazon-vpc-nat-gateway/config.yml

```
@@ -0,0 +1,37 @@
+slug: amazon-vpc-natgateway
+description: |
+  ## Monitor your NAT Gateway in New Relic's observability platform
+  Our NAT Gateway integration reports data from your NAT Gateway service to your New Relic account using AWS CloudWatch Metric Streams.
+  ## Get started!
+  This integration uses [AWS CloudWatch Metric streams](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/aws-metric-stream/) to get NAT Gateway traffic data into New Relic.
+  ## About this integration
+  With this integration you can monitor key traffic information about your NAT Gateway.
+  - Track your connection established rate
+  - View idle connection timeouts
+  - Track Byte traffic
+  - Monitor your active connections over time
+  - Check for packet drops or port allocation errors over time
+summary: |
+  Use the New Relic - Amazon VPC NAT Gateway Integration to monitor traffic to and from your NAT Gateway.
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic
+  - Joseph Counts
+title: Amazon VPC NAT Gateway
+documentation:
+  - name: Amazon VPC NAT Gateway integration docs
+    description: |
+      Monitor your NAT Gateway traffic using AWS CloudWatch Metric Streams
+    url: https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-nat-gateway-monitoring-integration/
+keywords:
+  - aws
+  - amazon
+  - nat gateway
+  - metrics
+  - vpc
+  - network
+installPlans:
+  - aws-cloudwatch-metric-streams
```

We actually have a data source for this! 
```suggestion
dataSourceIds:
  - amazon-cloudwatch-metric-streams
```